appbuilder/services/livesync/android-livesync-service.ts

```diff
@@ -25,8 +26,17 @@ export class AppBuilderAndroidLiveSyncService extends AndroidLiveSyncService imp
 		}).future<void>()();
 	}
 
-	public removeFiles(): IFuture<void> {
-		return Future.fromResult();
+	public removeFiles(appIdentifier: string, localToDevicePaths: Mobile.ILocalToDevicePathData[], deviceProjectRootPath?: string): IFuture<void> {
+		return (() => {
+			let commands =_.map(localToDevicePaths, ldp => {
+				let relativePath = path.relative(deviceProjectRootPath, ldp.getDevicePath()),
```

Comment:
`deviceProjectRootPath` is optional and `path.relative` will throw an exception if it is `undefined`.